src/IssueViz.Security/HotspotsList/TableDataSource/HotspotsTableDataSource.cs

```diff
@@ -0,0 +1,79 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Shell.TableManager;
+using SonarLint.VisualStudio.Core;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableDataSource
+{
+    internal interface IHotspotsTableDataSource : ITableDataSource, IDisposable
```

Comment:
Just an observation: the equivalent `ISonarErrorListDataSource' interface doesn't directly expose the `ITableDataSource`.
It depends what the purpose of this interface is for, but it's not providing any level of abstraction at present.

Reply:
I'll remove it for now